gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/task/DagProcessingEngineMetrics.java

```diff
@@ -162,6 +167,19 @@ public void markDagActionsConclude(DagActionStore.DagActionType dagActionType, b
     }
   }
 
+  // TODO: mark this metric when deleted
+  public void markDagActionsDeleted(DagActionStore.DagActionType dagActionType, boolean succeeded) {
```

Comment:
so why not using this methods?

Reply:
oh this is easier to implement than I initially realized. done